frontend/src/features/maps/routes/MapOverview.tsx

```diff
@@ -20,9 +22,12 @@ export default function MapOverview() {
   const { t } = useTranslation(['maps']);
   const [infoMessage, setInfoMessage] = useState(initialMessage);
 
+  const searchParams: MapSearchParameters = {
+    owner_id: getUser()?.profile.sub,
+  };
```

Comment:
Makes sense for this case, yes. But eventually we want to search for maps created by other users than the currently logged-in or no user at all. Maybe I split the backend endpoint into two seperate methods, instead of using one generic one for all possible use cases.

Reply:
Ah, I see, hence the PrivacyOptions enum. Then leave it like this^^

Reply:
I do see your point though, fetching maps for a page that has unlimited write priviliges with an openly accessible user_id poses a risk. We should think of this again as soons as we tackle security, privacy and priviliges on maps.